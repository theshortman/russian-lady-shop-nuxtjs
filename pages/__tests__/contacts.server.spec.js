/**
 * @jest-environment node
 */

import { renderToString } from '@vue/server-test-utils'
import ContactsPage from '../contacts.vue'

test('renders correctly contacts page', async () => {
  const str = await renderToString(ContactsPage)
  
  expect(str).toMatchSnapshot()
})
