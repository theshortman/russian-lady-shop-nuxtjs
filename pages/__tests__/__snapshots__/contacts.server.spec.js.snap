// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly contacts page 1`] = `"<div data-server-rendered=\\"true\\"><h1>Контакты</h1> <p>russkay-lady@yandex.ru</p></div>"`;
